.github/workflows/build_and_deploy.yml

```diff
@@ -86,22 +92,22 @@ jobs:
     name: Download & Deploy Artifact
     needs: build-and-upload
     runs-on: ubuntu-latest
+    # Working directory: '/home/runner/work/docs/docs'
     steps:
       - name: Download artifact
         uses: actions/download-artifact@v4
         with:
           name: html
+          # Downloads to '/home/runner/work/docs/docs/html'
           path: html
-      - name: show files
-        run: ls -lR
```

Comment:
I think @Czaki added this in #386 as a sanity check, so have removed, but happy to add back.